docs/glossary.md

```diff
@@ -0,0 +1,361 @@
+# Glossary
+
+---
+
+## A
+---
+
+#### Account
+
+Accounts allow you to keep separate records of your DCR funds. Transferring DCR across accounts will create a transaction on the blockchain.
+
+#### Address
+
+A secure identifier that represents a possible destination for a payment, similar to an email address. The address is a human-readable representation of the public key receiving payment.
+
+#### Atom
+
+The smallest unit of Decred currency. One atom is one hundred millionth of a single DCR (0.00000001 DCR).
+
+
+## B
+
+---
+
+#### Block
+
+The data structure transactions are bundled into before being written into the distributed ledger. Decred's difficulty adjusts such that new blocks are found every five minutes on average.
+
+#### Block explorer
+
+A tool for inspecting the contents of blocks in a more user-friendly format. 
+
+#### Block header
+
+Metadata at the beginning of a block that defines information about the block. The hash of a block header is the main way of identifying a block. 
+
+#### Block reward
+
+A reward (in DCR) split between PoW miners, ticket holders, and the Treasury for successfully approving a block of transactions. The block reward [decays exponentially](https://docs.decred.org/advanced/inflation/) with block height (length of blockchain). However, Decred’s algorithm interpolates this decay over time so as not to produce market shocks, as typically seen in algorithms that involve large and infrequent reductions, such as halving the rewards every 4 years. The block reward started at 31.19582664 and it adjusts every 6,144 blocks (approximately 21.33 days) by reducing by a factor of 100/1011.
+
+#### Block voting
+
+In each block, five tickets are called to vote. In addition to votes on any open consensus rule change proposals, each ticket votes to approve or reject the regular transaction tree of the previous block. If a majority of voting tickets vote No, the regular transaction tree of the previous block is rejected and those transactions are returned to the mempool.
```

Comment:
> If a majority of voting tickets vote No,

A majority must vote yes to approve, otherwise this results in disapproval (including if there is a tie).

Reply:
Updated to. Below. My only question (as a reader) is, if there are five votes, how is a tie possible? Can a ticket abstain from voting to approve transactions? In the docs, I see tickets can abstain vote on consensus rules, but I'm not seeing any reference to abstaining from voting on transactions. 

#### Block voting

In each block, five tickets are called to vote. In addition to votes on any open consensus rule change proposals, each ticket votes to approve or reject the regular transaction tree of the previous block. If a majority of voting tickets vote Yes, the regular transaction tree of the previous block is accepted. If a majority of voting tickets vote No (or there is a tie), the regular transaction tree of the previous block is rejected and those transactions are returned to the mempool.

Reply:
There can be four votes.  Only three votes are necessary to mine a block.